config/routes.rb

```diff
@@ -5,6 +5,14 @@
     sessions: "users/sessions",
   }
 
+  as :user do
+    get "/induction_coordinator/registration/school_confirmation", to: "users/induction_coordinators/registrations#confirm_school"
+    get "/induction_coordinator/registration/check_email", to: "users/induction_coordinators/registrations#start_registration"
```

Comment:
This can definitely be tidied up with namespaces and scopes. I'll take a look this afternoon